renku/service/entrypoint.py

```diff
@@ -47,7 +47,7 @@
 
 logging.basicConfig(level=os.getenv("SERVICE_LOG_LEVEL", "WARNING"))
 
-if os.getenv("SENTRY_DSN"):
+if os.getenv("SENTRY_ENABLED").lower() == "true":
```

Comment:
Same as above

Reply:
the service has a `config.py` is that not used anymore? That way you can make sure to do it right in one place and use the proper boolean from `config` elsewhere and avoid the problem in the future.

Reply:
Right, good point. I forgot about that one